.github/actions/message/action.yml

```
@@ -0,0 +1,20 @@
+name: "Status Check"
+description: "Produce configuration from commit message."
+
+inputs:
+  token:
+    description: "The github token."
+    required: true
+outputs:
+  build:
+    description: "If trigger build."
+  cache:
+    description: "If enable cache."
+  check:
+    description: "If trigger check."
+  macos:
+    description: "If enable macos."
+
+runs:
+  using: "node20"
+  main: "main.js"
```

It is not too YAML-ish in my opinion 😄 
```suggestion
name: Status Check
description: Produce configuration from commit message.

inputs:
  token:
    description: The github token.
    required: true
outputs:
  build:
    description: If trigger build.
  cache:
    description: If enable cache.
  check:
    description: If trigger check.
  macos:
    description: If enable macos.

runs:
  using: node20
  main: main.js
```